.github/workflows/teamcity-services-diff-check.yml

```diff
@@ -0,0 +1,93 @@
+name: TeamCity Services Diff Check
+permissions: read-all
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/teamcity-services-diff-check.yml'
+      - 'mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt'
+      - 'mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt'
+      - 'mmv1/products/**'
+jobs:
+  check-pr:
+    runs-on: ubuntu-22.04
+    outputs:
+      services: ${{steps.services.outputs.services}}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - id: services
+        run: |
```

Comment:
Could you add a `name` property here to help indicate what the step is doing, e.g. "Check for new services"